### PR TITLE
DIGREPO-466

### PR DESCRIPTION
**Pull request by dunn**
Originally at https://github.library.ucsb.edu/ADRL/alexandria/pull/2

### CODE REVIEW

File: app/models/solr_document.rb [hunk 1]
```diff
@@ -39,7 +39,9 @@ def curation_concern?
 
   # blank if there is no embargo or the embargo status
   def after_embargo_status
-    return if self['visibility_after_embargo_ssim'].blank?
+    return if self['visibility_after_embargo_ssim'].blank? ||
```
Comment (jcoyne) [inline app/models/solr_document.rb]: Please add a test for this situation.

Comment (dunn) [inline app/models/solr_document.rb]: I can't reproduce the error anymore, so I removed it.

**Comment by jcoyne** [comment]
@val99erie Can you review this PR?

File: app/services/update_metadata_from_proquest_file.rb [hunk 2]
```diff
@@ -33,7 +33,7 @@ def attributes
 
   def embargo_start_date
     if attributes[:DISS_agreement_decision_date].blank?
-      transformed_start_date
+      transformed_start_date(attributes)
```
Comment (val99erie) [inline app/services/update_metadata_from_proquest_file.rb]: You shouldn't need to change these methods to pass the attributes hash around.  There is an attributes ivar that already contains them, so you should just be able to directly call the attributes method whenever you need them.  The attributes method is here:
https://github.com/curationexperts/alexandria-v2/blob/master/app/services/update_metadata_from_proquest_file.rb#L27-L32

Comment (dunn) [inline app/services/update_metadata_from_proquest_file.rb]: That change/commit was only for the sake of readability, but I'll drop it from the PR.

**Comment by val99erie** [comment]
Looks good

**Pull request merged**